src/lesson5/Main.java

```diff
@@ -0,0 +1,143 @@
+package lesson5;
+
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+
+    protected static final int SIZE = 10000000;
+    protected static final int HALF = SIZE / 2;
+    protected static float[] arr1 = new float[SIZE];
+    protected static float[] arr2 = new float[SIZE];
+    protected static float[] arr2_1 = new float[HALF];
+    protected static float[] arr2_2 = new float[HALF];
+
+    public static void main(String[] args) throws Exception {
+
+        Thread thr1 = new Thread(new myCalculate1());
+        thr1.start();
+
+        Thread thr2 = new Thread(new myCalculate2());
+        thr2.start();
+    }
+
+    static class myCalculate1 extends Thread implements Runnable {
+
+        @Override
+        public void run() {
+            try {
+                // заполнени массива единицами
+                fillingArr(arr1);
+                // вычисление элементов массива по формуле
+                long start = System.currentTimeMillis();
+                CalculateArr(arr1, SIZE);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.